app/enterprise_test.go

```diff
@@ -116,6 +116,7 @@ func TestSAMLSettings(t *testing.T) {
 			}
 
 			th.Server.initEnterprise()
+			th.App.initEnterprise()
```

Comment:
Hmm .. wondering if we can do the server init inside the app init ? The `srv` field is inside App anyways.

Reply:
For testing, makes sense.

Reply:
I prefer to keep the separated, at least for now. The server is initialized a the launch of the application and the app, at the first app instantiation, so are two different moments.

Reply:
Ok